openstack/neutron/templates/etc/_ratelimit.yaml.tpl

```diff
@@ -37,12 +37,6 @@ groups:
 {{- end }}
 
 {{- if .Values.rate_limit.rates }}
-rates:
-{{- range $level_k, $level := .Values.rate_limit.rates }}
-  {{ $level_k }}:
-        {{- range $target_uri_type, $target_uri_type_limits := $level }}
-    {{ $target_uri_type }}:
-{{ $target_uri_type_limits | toYaml | indent 6 }}
-        {{- end }}
-{{- end }}
+rates: 
+{{ .Values.rate_limit.rates | toYaml | indent 3 }}
```

Comment:
Why do you use indent 3 here? This is a key on the second indention level. so I think `indent 2` would be fine.